docs/reference/search/run-a-search.asciidoc

```diff
@@ -0,0 +1,285 @@
+sorts these matching documents by `_score`, a <<relevance-scores,relevance
+score>> that measures how well each document matches the query.
+
+[source,console-result]
+----
+{
+  "took": 1,
+  "timed_out": false,
+  "_shards": {
+    "total": 1,
+    "successful": 1,
+    "skipped": 0,
+    "failed": 0
+  },
+  "hits": {
+    "total": {
+      "value": 3,
+      "relation": "eq"
+    },
+    "max_score": 0.9983525,
+    "hits": [
+      {
+        "_index": "user_logs_000001",
+        "_id": "2",
+        "_score": 0.9983525,
+        "_source": {
+          "@timestamp": "2020-12-07T11:06:07.000Z",
+          "user": {
+            "id": "8a4f500d"
+          },
+          "message": "Login successful"
+        }
+      },
+      {
+        "_index": "user_logs_000001",
+        "_id": "3",
+        "_score": 0.49917626,
+        "_source": {
+          "@timestamp": "2020-12-07T11:07:08.000Z",
+          "user": {
+            "id": "l7gk7f82"
+          },
+          "message": "Logout successful"
+        }
+      },
+      {
+        "_index": "user_logs_000001",
+        "_id": "1",
+        "_score": 0.42081726,
+        "_source": {
+          "@timestamp": "2020-12-06T11:04:05.000Z",
+          "user": {
+            "id": "vlb44hny"
+          },
+          "message": "Login attempt failed"
+        }
+      }
+    ]
+  }
+}
+----
+// TESTRESPONSE[s/"took": 1/"took": "$body.took"/]
+====
+
+[discrete]
+[[search-multiple-indices]]
+=== Search multiple indices
+
+To search multiple indices, add them as comma-separated values in the search API
+request path.
+
+.*Example*
+[%collapsible]
+====
+The following request searches the `user_logs_000001` and `user_logs_000002`
+indices.
+
+[source,console]
+----
+GET /user_logs_000001,user_logs_000002/_search
+{
+  "query": {
+    "match": {
+      "message": "login successful"
+    }
+  }
+}
+----
+// TEST[continued]
+// TEST[s/^/PUT user_logs_000002\n/]
+====
+
+You can also search multiple indices using an index pattern.
+
+.*Example*
+[%collapsible]
+====
+The following request uses the index pattern `user_logs*` in place of the index
+name. The request searches any indices in the cluster that start with
+`user_logs`.
+
+[source,console]
+----
+GET /user_logs*/_search
+{
+  "query": {
+    "match": {
+      "message": "login successful"
+    }
+  }
+}
+----
+// TEST[continued]
+====
+
+To search all indices in a cluster, omit the index name from the request path.
+Alternatively, you can use `_all` or `*` in place of the index name.
+
+.*Example*
+[%collapsible]
+====
+The following requests are equivalent and search all indices in the cluster.
+
+[source,console]
+----
+GET /_search
+{
+  "query": {
+    "match": {
+      "message": "login successful"
+    }
+  }
+}
+
+GET /_all/_search
+{
+  "query": {
+    "match": {
+      "message": "login successful"
+    }
+  }
+}
+
+GET /*/_search
+{
+    "query" : {
+        "match" : { "message" : "login" }
+    }
+}
+----
+// TEST[continued]
+====
```

Comment:
The two previous examples have a lead-in sentence that describes the collapsed example. Just noting that this example does not follow that convention. 

![image](https://user-images.githubusercontent.com/25848033/81006429-841e5e80-8e1d-11ea-9ada-adfeaeda887c.png)

Reply:
I think I'm okay with that in this case. Thanks for raising though!